src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -22,10 +22,12 @@ import './outline-step-view';
 import {css, customElement, html, LitElement, property, unsafeCSS} from 'lit-element';
 
 import {COMMON_STYLES} from './cloud-install-styles';
+import {ServerLocation} from '../../model/location';
+import {LOCATION_NAMES} from '../location_name';
 
 export interface Location {
   id: string;
-  name: string;
+  location: ServerLocation;
```

Comment:
It's preferable to have dump UI components that are not aware of the domain model. That makes the components reusable and easier to understand. I'd like to keep dependencies on the model in the CreateServerApp or anything else we designate as an "app".

Please revert to name. That can me a name_msg, so you can call localize on it in the HTML code.
Add a country field, which can be used to output the country name and derive the flag

Reply:
OK, I reorganized this so that the `DisplayLocation` type doesn't rely on the `GeoLocation` type.